Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -618,7 +618,9 @@ public IEnumerator RecordTransformValues(GameObject testObject)
 
                 // Two hand rotate/scale far
                 yield return rightHand.MoveTo(rightHandNearPos, numSteps);
+                yield return new WaitForFixedUpdate();
```

Comment:
Why do we need this here if it you doubled it in the moveTo?

Reply:
I removed that, I think I can remove this as well though since i think these didn't do anything.